src/frontend/src/lib/i18n/en.json

```diff
@@ -830,7 +830,8 @@
 			"btc_transactions": "BTC transaction information is obtained from central third parties and should be independently verified."
 		},
 		"warning": {
-			"no_index_canister": "Transaction for $token_list can not be displayed. No Index canister."
+			"no_index_canister": "Transaction for $token_list can not be displayed. No Index canister.",
+			"broken_index_canister": "Transaction for $token_list can not be displayed. Index canister is broken."
```

Comment:
unsolicited feedback: "...is broken" sounds scary. it does not feel like a word often used in the field. I would suggest to review the wording.

Reply:
@peterpeterparker I just talked with @StefanBerger-DFINITY and we decided that we change the text to "... is currently not available".

Reply:
Cool